doc/source/whatsnew/v0.20.0.txt

```diff
@@ -38,6 +38,25 @@ Other enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+-  ``map`` on an ``Index`` now returns an ``Index``, not an array (:issue:`12766`)
+.. ipython:: python
```

Comment:
Some sphinx sytax comments (sphinx is rather picky ..):

- blank line above `.. ipython ...`
- Can you also indent this line (and all lines below) with two spaces? (to be at the level of the content of the bullet point) -> so everything will be part of this bullet point